examples/wines.get.json

```diff
@@ -347,11 +347,57 @@
                                     "locale": "it",
                                     "value": "Expovina"
                                 }
-                            ]
+                            ],
+                            "sort_index": 10,
+                            "in_search": true
                         },
                         "year": 2016,
                         "wine_year": 2013,
-                        "valuation": "Gold"
+                        "valuation": {
+                            "type": "diploma",
+                            "text": [
```

Comment:
This is the text that can be used for fine-grained filtering when "Expovina" is selected.